plugins/woocommerce/tests/e2e-pw/tests/merchant/create-shipping-zones.spec.js

```diff
@@ -176,10 +180,17 @@ test.describe( 'WooCommerce Shipping Settings - Add new shipping zone', () => {
 				'flat_rate'
 			);
 			await page.click( '#btn-ok' );
+			await page.waitForLoadState( 'networkidle' );
+			await expect(
+				page
+					.locator( '.wc-shipping-zone-method-title' )
+					.filter( { hasText: 'Flat rate' } )
+			).toBeVisible();
```

Comment:
Adding wait and assertion here because of the same reasoning in https://github.com/woocommerce/woocommerce/pull/35760#discussion_r1041987217